scripts/bert/finetune_classifier.py

```diff
@@ -41,22 +41,25 @@
 import mxnet as mx
 from mxnet import gluon
 from gluonnlp.model import bert_12_768_12
-from bert import BERTClassifier
+from bert import BERTClassifier, BERTRegression
 from tokenizer import FullTokenizer
-from dataset import MRPCDataset, ClassificationTransform
+from dataset import MRPCDataset, QQPDataset, RTEDataset, \
+    STSBDataset, ClassificationTransform, RegressionTransform, \
+    QNLIDataset, COLADataset, MNLIDataset, WNLIDataset, SSTDataset
 
 np.random.seed(0)
 random.seed(0)
 mx.random.seed(2)
 logging.getLogger().setLevel(logging.DEBUG)
 
-parser = argparse.ArgumentParser(description='BERT sentence pair classification example.'
-                                             'We fine-tune the BERT model on MRPC')
+parser = argparse.ArgumentParser(description='BERT fine-tune examples for GLUE tasks.')
 parser.add_argument('--epochs', type=int, default=3, help='number of epochs')
 parser.add_argument('--batch_size', type=int, default=32,
                     help='Batch size. Number of examples per gpu in a minibatch')
 parser.add_argument('--test_batch_size', type=int, default=8, help='Test batch size')
 parser.add_argument('--optimizer', type=str, default='adam', help='optimization algorithm')
+parser.add_argument('--task_name', type=str, default='MRPC',
```

Comment:
adding `choices=tasks.keys()` might be useful

Reply:
Right, It's more robust and simple.